docker/ubuntu/baseimage/bootstrap.sh

```diff
@@ -7,12 +7,13 @@ dpkg --add-architecture i386
 apt-get update
 apt-get upgrade -y
 
-ARM_SF_TC=(gcc-arm-linux-gnueabi g++-arm-linux-gnueabi)
-ARM_HF_TC=(gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf)
-ARM64_TC=(gcc-aarch64-linux-gnu g++-aarch64-linux-gnu)
+ARM_SF_TC=(gcc-8-arm-linux-gnueabi g++-8-arm-linux-gnueabi)
+ARM_HF_TC=(gcc-8-arm-linux-gnueabihf g++-8-arm-linux-gnueabihf)
```

Comment:
Also drat, I didn't notice/realize this would cause some minor issues too (for sony-prstux). But that's minor and easily resolved.